src/bin/console/console.cpp

```diff
@@ -480,11 +487,7 @@ int Console::_load_table(const std::string& args) {
       // at some point.
       static constexpr auto DEFAULT_CHUNK_SIZE = 500'000u;
```

Comment:
We could set this to `100'000u`. Looking also at #1263, we have multiple locations where we set a chunk size either to a manually defined fixed size or to `Chunk::MAX_SIZE`. Currently, `100'000u` is the recommended chunk size. We could either introduce something like `Chunk::RECOMMENDED_SIZE` or find a solution for #63.